### PR TITLE
Use Puppet 4 in bats test runs

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by stbenjam** [comment]
Can we make it configurable so it can be set by the Jenkin's job's environment?

**Comment by ehelms** [comment]
I had a feeling you might ask for that.

**Comment by ehelms** [comment]
OK @stbenjam what do you think?

File: bats/fb-install-katello.bats [hunk 1]
```diff
@@ -73,11 +73,17 @@ setup() {
     cd katello-deploy-master
   fi
 
+  cmd = "ruby setup.rb --install-options=\"-v --foreman-admin-password=changeme --disable-system-checks\" --scenario katello" --koji-repos --scenario katello --puppet-four
```
Comment (stbenjam) [inline bats/fb-install-katello.bats]: I think `--puppet-four` needs to come off this line right?

Comment (ehelms) [inline bats/fb-install-katello.bats]: Truth. Fixed

**Comment by stbenjam** [comment]
APT

**Pull request merged**